Pod/Classes/CZPickerView.m

```diff
@@ -308,7 +308,11 @@ - (UITableViewCell *)tableView:(UITableView *)tableView cellForRowAtIndexPath:(N
             cell.accessoryType = UITableViewCellAccessoryCheckmark;
         }
     }
-    if ([self.dataSource respondsToSelector:@selector(czpickerView:attributedTitleForRow:)]) {
+    if([self.dataSource respondsToSelector:@selector(czpickerView:titleForRow:)] && [self.dataSource respondsToSelector:@selector(czpickerView:titleForRow:)]){
+        cell.textLabel.text = [self.dataSource czpickerView:self titleForRow:indexPath.row];
+        // add image to cell
+        cell.imageView.image = [self.dataSource czpickerView:self imageForRow:indexPath.row];
```

Comment:
I've tested with icons and large images from 200k(1206 × 800) to 900k(2560 × 1600), it seems there's no problem with large pictures:). In my case, I added 10 cells into the table, each with a large enough image. After picked, I present all that chosen in a scrollview. Tested on iP6, seems work correctly.
But in real situation, image here is more like "icon", normally won't be too large.